packages/common/src/domain/entities/tag.ts

```diff
@@ -0,0 +1,20 @@
+import { Entity, JoinColumn, JoinTable, ManyToMany, ManyToOne, PrimaryColumn } from "typeorm";
+import { Transaction } from "./transaction";
+ import { User } from "./user";
+
+ @Entity()
+ export class Tag {
```

Comment:
To be more specific we should probably prefix the tag with transaction?

```suggestion
 export class TransactionTag {
```